hack/dockerfiles/test.Dockerfile

```diff
@@ -98,6 +98,7 @@ COPY --from=buildkitd /usr/bin/buildkitd /usr/bin
 FROM buildkit-base AS integration-tests
 ENV BUILDKIT_INTEGRATION_ROOTLESS_IDPAIR="1000:1000"
 RUN apk add --no-cache shadow shadow-uidmap sudo \
+  && [ -d /var/mail ] || mkdir /var/mail \
```

Comment:
nit: `mkdir -p /var/mail` ?

Reply:
Both are okay I think, "/var" is there, no need "-p" option here.

Reply:
`-p` allows skipping `[ -d /var/mail ]` check

Reply:
ping @AkihiroSuda

Reply:
Other approach can be found here; https://github.com/docker/cli/pull/1728

/cc @ijc